package.json

```diff
@@ -29,6 +29,8 @@
     "@mapbox/mapbox-sdk": "^0.15.3",
     "@material-ui/core": "^1.5.1",
     "@material-ui/icons": "^1.1.1",
+    "@math.gl/core": "^4.0.1",
+    "@math.gl/web-mercator": "^4.0.1",
```

Comment:
why do we need new dependencies?

Reply:
from react-map-gl upgrade docs:

`viewport-mercator-project` (an alias of `@math.gl/web-mercator`) is no longer a dependency. You can still install the library on the side as a utility for viewport-related math, but it's no longer required.